app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+  before_action :find_user_group_team, only: %i[summary data]
+  before_action :initialize_presenter, only: %i[summary data]
+
+  def dashboard; end
+
+  def summary
+    respond_to do |format|
+      format.html
+      format.json do
+        dataset_labels = t("graphs.#{params[:graph_kind]}.dataset_labels.#{params[:time_scope] || 'weekly'}", default: nil)
+        units = t("graphs.#{params[:graph_kind]}.units", default: '')
+
+        render json: @presenter.to_json(
+          graphs: [{ type: :line_graph, data: :team_individual_data, units: units, dataset_labels: dataset_labels }]
+        )
+      end
+    end
+  end
+
+  def data; end
+
+  private
+
+  def find_user_group_team
+    # assume that user has only one team user group
+    @user_group = current_user.user_groups.joins(:group_type).where("group_types.name = 'team'").first
+  end
```

Comment:
We don't use @user_group anywhere in the user views. I get that it's needed for the TeamIndividualPresenter but perhaps a bit of refactoring is needed, as these pages shouldn't depend on the user being in a group.
(Also if they aren't in a group then end up with a nil:NilClass error.

Reply:
so we can not assume that:
1. user always belongs to at least one user group
2. user has always team (user group type team) set ?

Reply:
I don't think we can assume either of those things. 
A smaller organisation might use Murfin+ and not bother with any organisational structure if there are e.g. less than 100 users.  And we can't assume they have a group type with 'team' as the name either. The groups types could be anything, directorate, division, specialty etc.

Reply:
Make sense, I will address those assumptions in this PR next